styleguide/components/71-pages/general/news-overview-page/news-overview-page.twig

```diff
@@ -25,6 +25,7 @@
         "type": "overview-title",
```

Comment:
Pas je de naam aan naar news and events?

Reply:
Fixed.

Reply:
Ook in de config he :)

Reply:
Done